src/popup/utils/constants.ts

```diff
@@ -227,9 +210,6 @@ export const NODE_STATUS_CONNECTING = 'connecting';
 export const NODE_STATUS_CONNECTED = 'connected';
 export const NODE_STATUS_ERROR = 'error';
 
-export const AGGREGATOR_URL = 'https://superhero.com/';
-export const CONTACT_EMAIL = 'superherowallet@protonmail.com';
-
 export const TXS_PER_PAGE = 30;
 export const AENS_DOMAIN = '.chain';
 export const AENS_NAME_MAX_LENGTH = 63 + AENS_DOMAIN.length;
```

Comment:
These are Aeternity-specific constants but I don't think this should be moved to the 'protocol/aeternity/` directory. Since our app in general offers more features of Aeternity, we can move constants or methods to each specific protocol level directory which are only common for all the protocols but distinct for each one of them.

Reply:
The AENS is specific to Aeternity protocol and it definitely should be moved to the `protocols/aeternity/`.
Agreed on the call.